src/components/shareButton/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { SizeType } from 'antd/es/config-provider/SizeContext';
+import ShareMenu from '../shareMenu';
+import ShareWhite from '../../assets/images/sharewhite-small.png';
+
+const StyledShareButton = styled(Button)`
+  margin: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+`;
+
+const StyledImg = styled.img`
+  max-height: 25px;
+  margin-bottom: 4px;
+`;
+
+// link prop is primarily for facebook as the post can only be prefilled with a link, no body text
```

Comment:
Not everything should be represented as a single object. We have an enumeration here: one repr per social media. What does FI/FII tell us about how to design such data and behavior?

Reply:
I'm not familiar with FI/FII, could you please explain what it means? Thanks!

Reply:
FI and FII just mean Fundies 1 and 2